openapi/neutrino/types.bal

```diff
@@ -17,1029 +17,1029 @@
 # Map containing timezone details
 public type Timezone record {
     # The time zone ID as per the IANA time zone database (tzdata)
-    string? id;
+    string id;
```

Comment:
Fields are no longer nillable. Could you please check ?

Reply:
There is no `nullable: true` for the `id of the timezone schema` in the openapi.yml. This is a change to the generated client.bal. so even `--nullable` is tried, no change happened.

Reply:
Please try `bal openapi -i openapi.yml --mode client --nullable`. All the fields should be nullable according to the original connector.